Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -239,23 +242,32 @@ private struct ChangeStreamTestFile: Decodable {
 /// Class covering the JSON spec tests associated with change streams.
 final class ChangeStreamSpecTests: MongoSwiftTestCase {
     func testChangeStreamSpec() throws {
-        let tests = try retrieveSpecTestFiles(specName: "change-streams", asType: ChangeStreamTestFile.self)
+        let tests = try retrieveSpecTestFiles(
+            specName: "change-streams",
+            subdirectory: "legacy",
```

Comment:
tests in the old format were moved into the legacy directory.

@bynn, for some context, we used to have custom YAML/JSON formats for a number of specifications' tests. this was not great because new test runners take a lot of work to implement. we recently created a "unified" format that we are moving toward using across all specs so it is easier for us to test that drivers across languages have consistent behavior.

however, not all tests have been ported to the new format yet so we still have to run some of them.